packages/common/src/ui/layout/tables/context/TableContext.test.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { FC } from 'react';
-import { TableStore, useIsGrouped } from './TableContext';
 import { act } from 'react-dom/test-utils';
 import { renderHook } from '@testing-library/react-hooks';
-import { TableProvider, createTableStore } from './TableContext';
+import { useIsGrouped } from './hooks';
```

Comment:
uhh.. very strange - the only time I touched this file was to update this import